src/pages/gen2/build-a-backend/data/customize-authz/signed-in-user-data-access/index.mdx

```diff
@@ -47,4 +68,24 @@ const schema = a.schema({
 });
 ```
 
+In your application, you can perform CRUD operations against the model using `client.models.<ModelName>` with the `iam` auth mode. Note in this instance the user must be authenticated for the client to sign the request using the Amplify-generated `AuthRole` IAM role.
```

Comment:
same feedback as above, in this case, we should probably direct customers to the fact that they must've been signed in on their auth category. Maybe cross-link